boards/BMS/Inc/App/configs/App_PreChargeThreshold.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#define MIN_PRE_CHARGE_DURATION_MS (1110U)
+#define MIN_PRE_CHARGE_COMPLETE_DURATION_MS (1850U)
```

Comment:
So `MIN_PRE_CHARGE_DURATION_MS` is the time it takes to charge capacitors to 90% of accumulator voltage. Does `MIN_PRE_CHARGE_COMPLETE_DURATION_MS` refer to the time it takes to charge them to 98%, and `MAX_PRE_CHARGE_COMPLETE_DURATION_MS` is a value slightly above that?

Reply:
Yes for `MIN_PRECHARGE_DURATION_MS`. `MIN/MAX_PRE_CHARGE_COMPLETE_DURATION_MS` are the lower and upper thresholds for precharge completion